docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3'
+
+volumes:
+  postgres_data: {}
+  postgres_backup: {}
+
+services:
+  db:
+    container_name: andela_social_db
```

Comment:
*tip:* it would be good to namespace your container name by the environment in which they operate. 
*local, staging, prod* etc

Reply:
It would be good to do this by passing the container name also from the environment variable 
`container_name: ${CONTAINER_NAME}_{ENVIRONMENT}`

Reply:
We maintain what you already have. The change I am requesting would involve creating an environment file and adding that to your various docker-compose services. 

You can then do this ${CONTAINER_NAME} to read your environment value for container name into the docker compose file.